src/DataTable/index.tsx

```diff
@@ -215,6 +217,11 @@ export const DataTable = <T extends Record<string, unknown>>(
   // If the incoming data changes, override the table data
   useEffect(() => {
     setData(data)
+    if(data.length == 0 && isEditing){
```

Comment:
This is going to set the first row as editing even if isEditing is false if the table has data in it. We also want to include isEditing in the dependencies for this useEffect call.

Reply:
Oh! I did not realize this, sorry!!